CentralComputing/Utils.cpp

```diff
@@ -40,20 +41,20 @@ void Utils::print(LogLevel level, const char * format, ...){
       }
 }
 
-int Utils::toogleGPIOUp(int GPIONumber) {
-    std::string start = "/sys/class/gpio/gpio"
+int Utils::set_GPIO_high(int GPIONumber) {
+    std::string start = "/sys/class/gpio/gpio";
     std::string integer = std::to_string(GPIONumber);
-    std::string end = "/value"
+    std::string end = "/value";
     std::string path = start + integer + end;
     std::ofstream out(path);
     out<<"1";
     out.close();
     return 1; //Have it return 1 if it works and zero otherwise
 }
-int Utils:toggleGPIODown(int GPIONumber) {
-    std::string start = "/sys/class/gpio/gpio"
+int Utils::set_GPIO_low(int GPIONumber) {
```

Comment:
The code for set_GPIO_high and set_GPIO_low is the same except for what you write, either "0" or "1". Is there a way to easily combine these functions together? Consider an additional boolean input.... Should also simplify the `counter` in the Pod.cpp loop